.github/workflows/ci_python.yml

```diff
@@ -0,0 +1,175 @@
+# This file is autogenerated by maturin v1.2.0
+# To update, run
+#
+#    maturin generate-ci github --pytest --zig -o ../.github/workflows/ci_python.yml
+#
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - master
+    tags:
+      - '*'
+  pull_request:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target: [x86_64, x86, aarch64, armv7, s390x, ppc64le]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
```

Comment:
We may want to have a step after this that runs tests (and type-checking, etc.) in multiple versions.